### PR TITLE
Append proper screen to checkbox and radio element

### PR DESCRIPTION
**Pull request by korneliakobiela**
Problem: Docs had an only reference image to radio element
Solution: Add checkbox element image to documentation

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

### Change Description ###

Append images and buttons to tau documentation.

### CODE REVIEW

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm [hunk 1]
```diff
@@ -33,8 +33,14 @@ <h1>Checkbox</h1>
...
 			window.open('https://code.tizen.org/demos?path=1.0%2Fexamples%2Fmobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fcheckbox.html', '_blank')"
->Use example</button>
-
+>Use checkbox example</button>
+<button
+	onclick="
+		if (navigator.userAgent.toLowerCase().indexOf('chrome') == -1) {
+			alert('Chrome Browser required')
+		} else
+			window.open('https://code.tizen.org/demos?path=1.0%2Fexamples%2Fmobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fradio.html', '_blank')"
```
Comment (lslachciak) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Checkbox.htm]: mobile -->wearable

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm [hunk 2]
```diff
@@ -13,8 +13,8 @@ <h1>Checkbox and Radio Button</h1>
 
 <p>Checkbox and Radio Button components shows selectable items:</p>
 
-<img src="../../images/checkboxRadio.png" alt="photo of checkboxRadio in Tizen">
-
+<img src="../../images/checkboxRadio.png" alt="photo of radio in Tizen">
+<img src="../../images/checkbox-wearable.png" alt="photo of checkbox in Tizen">
```
Comment (singa2000) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm]: Would you add the sample link button, please?

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm [hunk 3]
```diff
@@ -13,8 +13,8 @@ <h1>Checkbox and Radio Button</h1>
 
 <p>Checkbox and Radio Button components shows selectable items:</p>
 
-<img src="../../images/checkboxRadio.png" alt="photo of checkboxRadio in Tizen">
-
+<img src="../../images/checkboxRadio.png" alt="photo of radio in Tizen">
```
Comment (singa2000) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm]: Is it possible add the  image title? For which is radio button, which is checkbox..

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm (hunk 2 above)
Comment (lslachciak) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm]: @singa2000 sample link was in 5.0 version of docs, I moved it to 5.5 too

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm (hunk 3 above)
Comment (lslachciak) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm]: @singa2000 I added proper titles

**Comment by lslachciak** [comment]
Improved PR according to the @singa2000 comments. Please check stg urls:
https://docs.stg.tizen.org/staging/981/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm
https://docs.stg.tizen.org/staging/981/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

**Comment by mijin85cho** [comment]
Thanks for the update. To merge this PR, I hope cross check and approval from one of you team member.

**Comment by lslachciak** [comment]
@singa2000 could you approve?

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm [hunk 4]
```diff
@@ -13,23 +13,29 @@ <h1>Checkbox and Radio Button</h1>
 
 <p>Checkbox and Radio Button components shows selectable items:</p>
 
+<h2>Checkbox</h1>
 <img src="../../images/checkbox-wearable.png" alt="photo of checkboxRadio in Tizen">
```
Comment (TomaszLukawskiSam) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm]: "checkboxRadio" - I think this is bug. I think should be "photo of Checkbox and Radio button in Tizen"

Comment (lslachciak) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm]: fixed

**Comment by lslachciak** [comment]
Improved PR according to the @TomaszLukawskiSam comments
Please check stg urls:
https://docs.stg.tizen.org/staging/981/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm
https://docs.stg.tizen.org/staging/981/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

**Comment by lslachciak** [comment]
@TomaszLukawskiSam could you re-check/re-approve ?

**Comment by lslachciak** [comment]
Rebased PRs on the latest master
@TomaszLukawskiSam Please re-check stg PRs

https://docs.stg.tizen.org/staging/981/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm
https://docs.stg.tizen.org/staging/981/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

**Pull request merged**